docs/src/main/asciidoc/_includes/devtools/create-app.adoc

```diff
@@ -52,13 +52,12 @@ ifndef::devtools-no-gradle[]
 To create a Gradle project, add the `--gradle` or `--gradle-kotlin-dsl` option.
 endif::[]
 
-_For more information about how to install the Quarkus CLI and use it, please refer to xref:cli-tooling.adoc[the Quarkus CLI guide]._
+For more information about how to install and use the Quarkus CLI, see the xref:cli-tooling.adoc[Quarkus CLI] guide.
 ****
-
```

Comment:
I'm not sure this is a good idea at all. I would rather keep the blank line.

Why did you remove it?

Reply:
The inclide acts as a unit. If I want to glue that unit to something else, for example to a numbered procedural step, I need to use `+` in the empty lines between the step and the include. IF the include ends with an empty row, the `+` glue will be not effective and will not do the job, not aling the contet, and the numbering sequence will be broken. This mean that the next numbered step, which should be 2, will be one again. Thats how asciidoc works.

Here is one example (just for the demo):
![image](https://github.com/quarkusio/quarkus/assets/48474054/aa86e28f-7d8b-4749-9e32-84cc4f47ad1d)

![image](https://github.com/quarkusio/quarkus/assets/48474054/f0b581b7-6c94-4598-997b-d3f01573cc9a)

I find out that we can work around that by wrapping the whole include in the `====` blocks. Then the empty line could be there, but if somebody would like to, for example, stick a NOTE to a snippet, and the note should not be the part of the snippet, we need to use `+` to align it properly. If the snipped will be ending with empty line and the whole thing around the `::include` will be not wrapped, it will not work.

Reply:
Oh, my apologies. I thought you are pointing to the lines deleted at the end of the file. Yeah, I think I will put these back.